platform/commonUI/general/src/directives/MCTSplitPane.js

```diff
@@ -93,13 +93,22 @@ define(
          * @memberof platform/commonUI/general
          * @constructor
          */
-        function MCTSplitPane($parse, $log, $interval) {
+        function MCTSplitPane($parse, $log, $interval, $window) {
             function controller($scope, $element, $attrs) {
                 var anchorKey = $attrs.anchor || DEFAULT_ANCHOR,
+                    positionParsed = $parse($attrs.position),
                     anchor,
                     activeInterval,
-                    positionParsed = $parse($attrs.position),
-                    position; // Start undefined, until explicitly set
+                    position,
+                    splitterSize,
+
+                    //create a unique alias for every instance with alias specified
+                    alias = $attrs.alias !== undefined ?
+                      "mctSplitPane-" + $attrs.alias : undefined,
+
+                    //convert string to number from localStorage
```

Comment:
It's worth noting that this is a good comment (where others are unnecessary).  This comment explains "why," where other comments explain "what".  More importantly-- this explains something that is not immediately clear to others--
 localStorage stores all values as strings.